src/architecture/utilities/keplerianOrbit.cpp

```diff
@@ -33,7 +33,6 @@ KeplerianOrbit::KeplerianOrbit()
     this->right_ascension = 0.0;
```

Comment:
Clean up the commit message to remove 'Former-commit-id:...`

Reply:
Done.